source/index.js

```diff
@@ -13,6 +13,7 @@ renderer.roundPixels = true
 
 document.getElementById("mount").appendChild(renderer.view)
 window.game = new Pixi.Container()
+game.playerCount = 0
 game.addChild(new Junkership())
```

Comment:
That'll work. I think maybe we could push it out to it's own file. As more stuff gets piled into it, it'll help keep index.js readable. I'll start pulling that out.